x-pack/plugins/security_solution/public/common/components/discover_in_timeline/use_discover_in_timeline_actions.tsx

```diff
@@ -40,17 +37,11 @@ export const useDiscoverInTimelineActions = (
   const { addError } = useAppToasts();
 
   const {
-    services: {
-      customDataService: discoverDataService,
```

Comment:
we need to keep discoverDataService and savedSearchService, used elsewhere below

Reply:
they're not removed, just on the same line. I only removed dataViewService